services/environments/snd/base/kustomization.yaml

```diff
@@ -9,6 +9,7 @@ resources:
 - ../../../adp-platform/kustomize.yaml
 - ../../../eutd/eutd-trade-exports-ai/base/patch
 - ../../../ffc/ffc-find/base/patch
+- ../../../coreai/coreai-mcu/base/patch
```

Comment:
this is redundant

Reply:
@DaRoszko  we shall remove it when we have migrated all the coreai services to the new namespace.

Reply:
It Is removed.